plugins/smart_proxy_ansible/links

```diff
@@ -1 +1,2 @@
 var/lib/foreman-proxy/ansible usr/share/foreman-proxy/.ansible
+usr/bin/ansible-runner-environment.sh usr/libexec/foreman-proxy/ansible-runner-environment
```

Comment:
I don't think that file should be in `/usr/bin` under that name? it feels confusing to have it names `ansible-runner-<something>` while in reality it is foreman-proxy specific

Reply:
Do we want in in usr/bin under some name or should I just move it out of the way?

Reply:
Probably not, they might use it for "run ansible as foreman would", but considering they still need to provide a path to the environment file, we probably don't need to lose any sleep over it.

Reply:
Yeah, I think libexec-only is fine. For those who will want "run ansible as foreman" can do that from there.